webapp/components/survey/CategorySelector/CategorySelector.js

```diff
@@ -45,25 +54,40 @@ const CategorySelector = (props) => {
           type="button"
           className="btn btn-s"
           style={{ justifySelf: 'center' }}
-          onClick={async () => {
-            const newCategory = await dispatch(createCategory(history, analysis))
-            onChange(newCategory)
-          }}
+          onClick={onAdd}
           aria-disabled={disabled}
         >
           <span className="icon icon-plus icon-12px icon-left" />
           {i18n.t('common.add')}
         </button>
       )}
+      {showCategoryPanel && (
```

Comment:
add category and categories panel next to each other

Reply:
ok